spec/features/trainees/diversities/diversity_progress_spec.rb

```diff
@@ -16,6 +16,7 @@
   scenario "renders an 'in progress' status when diversity information partially provided" do
     given_valid_diversity_information
     when_i_visit_the_diversity_section
+    and_i_am_directed_to_the_diversity_confirmation_page
```

Comment:
i reckon you could get rid of the line above `when_i_visit_the_diversity_section` as `and_i_am_directed_to_the_diversity_confirmation_page ` is heading straight to the page anyway but i think it should then be renamed to something like `when_i_visit_the_diversity_confirmation_page`

Reply:
cool, thanks. Changes made.